frontend/packages/dev-console/src/components/topology/TopologyPage.tsx

```diff
@@ -77,7 +77,7 @@ const TopologyPage: React.FC<Props> = ({ match, activeApplication, knative, cheU
       <Helmet>
         <title>Topology</title>
       </Helmet>
-      <NamespacedPage>
+      <NamespacedPage topologyView>
```

Comment:
You should pass in a `toolbar` or something here with the toggle button. This component should check the URL and render either the list or topology as a child.